src/Kiota.Builder/Writers/Python/PythonConventionService.cs

```diff
@@ -126,7 +126,7 @@ private static string TranslateInternalType(CodeType type)
             "void" => "None",
             "DateTimeOffset" => "datetime",
             "boolean" => "bool",
-            "Object" or "object" or "float" or "bytes" or "datetime" or "timespan" => type.Name,
+            "Object" or "object" or "float" or "bytes" or "datetime" or "timedelta" or "date" or "time" => type.Name,
```

Comment:
Same here.

Reply:
Updated in https://github.com/microsoft/kiota/pull/2454/commits/5ffbb02944387e0cce7afe7fd2bfee27da159801